R/private-lib.R

```diff
@@ -166,7 +190,7 @@ download_private_lib <- function(quiet = FALSE) {
   l <- lock_private_lib(lib, download = TRUE)
   on.exit(unlock_private_lib(l), add = TRUE)
   if (!is.null(pkg_data$remote)) pkg_data$remote$kill()
-  pkg_data$deps <- pkg_data$deps %||% lookup_deps("pkg")
+  pkg_data$deps <- pkg_data$deps %||% lookup_deps(.packageName, .libPaths())
```

Comment:
I think this might have been a bug left over from the rename of the package from 'pkg' to 'pak'.